posthog/test/base.py

```diff
@@ -67,6 +67,23 @@ def _setup_test_data(klass):
         klass.organization_membership = klass.user.organization_memberships.get()
 
 
+class FuzzyInt(int):
```

Comment:
We should definitely have a comment about why we made this wonderful thing
```suggestion
class FuzzyInt(int):
"""
Some query count assertions vary depending on the order of tests in the run because values are cached and so their related query doesn't always run.

For the purposes of testing query counts we don't care about that variation
"""
```